ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -161,9 +163,11 @@ enum PanelContent {
   Create = 'Create',
   Customize = 'Customize',
   Delete = 'Delete',
+  Disabled = 'Disabled',
   Confirm = 'Confirm'
 }
 
+// this is only used in the test.
```

Comment:
is this comment still needed?

Reply:
Probably not, but I'd like to leave it there because it confused me for a good five minutes that this was here and then not used in this file